react/components/molecules/key-value-placeholder/key-value-placeholder.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { withKnobs, number } from "@storybook/addon-knobs";
+
+import { KeyValuePlaceholder } from "./key-value-placeholder";
+
+storiesOf("Atoms", module)
+    .addDecorator(withKnobs)
+    .add("Key Value Placeholder", () => {
+        const numberOfLines = number("Number of lines", 1);
+        const placeholderColors = ["#DBDEE5", "#E3E6ED", "#DBDEE5"];

```

Comment:
Use lowercased hexadecimal please

Reply:
Vscode was claiming that lowercase was wrong, I'm gonna fix this and my configs.